Dockerfile

```diff
@@ -0,0 +1,85 @@
+FROM ubuntu:20.04
+
+ENV TERM="xterm-color"
+ARG USER_ID=999
+ARG GROUP_ID=999
+ARG USER=docker
+ARG PWD=docker
+ARG HOME="/home/$USER"
+
+# Create our group and user so we can login as ourself. Useful for ssh.
+RUN groupadd -g $GROUP_ID $USER && \
+    useradd -g $GROUP_ID -m -s /bin/bash -u $USER_ID $USER
+
+# Suppresses prompt for timezone
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Yocto dependencies
+RUN apt-get update
+RUN apt-get -y upgrade
+RUN apt-get -y install gawk make wget tar bzip2 gzip python3 unzip perl patch \
+    diffutils diffstat git cpp gcc texinfo chrpath \
+    ccache socat \
+    python3-pexpect findutils file cpio python python3-pip \
+    python3-jinja2 xterm bash
```

Comment:
You should add 'RUN apt autoremove -y' which will clean up old package data that is no longer needed.